.travis.yml

```diff
@@ -15,8 +15,8 @@ env:
     - PYTHONFAULTHANDLER=x
     - PYTHONWARNINGS=all
   matrix:
-    - DOCUTILS=0.11
```

Comment:
At this PR, I removed docutils-0.11 from test matrix to prevent to increase test time.
@shimizukawa what do you think?